doc-content/enterprise-only/cases/case-management-subcases-con.adoc

```diff
@@ -29,7 +27,7 @@ Optional data access restrictions where `XXX` is the name of the data item and t
 DestroyOnAbort::
 Optional indicator that tells the {PROCESS_ENGINE} whether to cancel or destroy the subcase when the subcase activity is aborted. The default value is `true`.
 UserRole_XXX::
-Optional user to case role mapping. You can reference the case instance role names here, meaning that an owner of the main case can be mapped to an owner of the subcase. The person assigned to main case is automatically assigned to the subcase, where `XXX` is the role name and the value of the property is the value of the group role assignment.
+Optional user to case role mapping. You can reference the case instance role names here, meaning that an owner of the main case can be mapped to an owner of the subcase. The person assigned to the main case is automatically assigned to the subcase, where `XXX` is the role name and the value of the property is the value of the group role assignment.
```

Comment:
Since we talk here about UserRole, the end of the last sentence should be: `the value of the property is the value of the user role assignment`

Reply:
Corrected as suggested.